sys-libs/libxcrypt/libxcrypt-4.4.28-r1.ebuild

```diff
@@ -196,6 +197,16 @@ multilib_src_configure() {
 		*) die "Unexpected MULTIBUILD_ID: ${MULTIBUILD_ID}";;
 	esac
 
+	tc-export PKG_CONFIG
+
+	if is_cross; then
+		if tc-is-clang; then
```

Comment:
tc-is-clang will check the active compiler for CHOST. It seems flawed to assume we should use same compiler (clang or gcc) for CTARGET.

This hack is ok for now I guess. To do this properly, I think we would need to introduce a new function in toolchain-funcs.eclass to check for commands prefixed with CTARGET, similar to tc-getBUILD_PROG.